docs/overview/office-add-ins-community-call.md

```diff
@@ -11,7 +11,8 @@ The Microsoft Office Add-ins community call is a monthly call where you can lear
 
 ## Agenda for February 9th call
 
-The agenda for February will be posted about a week before the call.
+- Work with data types in the Excel API
```

Comment:
Consider linking to the overview of the new data types feature. Also, long items in a bulleted list should end with a period ([Style guide article](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28751)).
```suggestion
- Work with [data types](../excel/excel-data-types-overview.md) in the Excel API.
```